scipy/stats/_hypotests.py

```diff
@@ -141,46 +141,59 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
     return Epps_Singleton_2sampResult(w, p)
 
 
-PoissonMeansTestResult = namedtuple('PoissonMeansTestResult', ('statistic', 'pvalue'))
```

Comment:
We have been avoiding `namedtuples` because they are harder to extended in the future. People use them as iterables and if they expect a different number of elements than they get, it breaks code.

Reply:
ok, i don't have any specific reason when choosing `namedtuples` other than following `Epps_Singleton_2sampResult = namedtuple`